Sources/DeepLearning/Layer.swift

```diff
@@ -1331,15 +1330,15 @@ public struct SimpleRNNCell<Scalar: TensorFlowFloatingPoint>: RNNCell, VectorNum
     @differentiable
     public func call(_ input: Input) -> Output {
         let concatenatedInput = input.input.concatenated(with: input.state, alongAxis: 1)
-        let newState = tanh(matmul(concatenatedInput, weight) + bias)
+        let newState = tanh(matmul(concatenatedInput, W_h) + b_h)
         return Output(output: newState, state: newState)
     }
 }
 
 /// An LSTM cell.
 public struct LSTMCell<Scalar: TensorFlowFloatingPoint>: RNNCell, VectorNumeric {
-    public var inputWeight, updateWeight, forgetWeight, outputWeight: Tensor<Scalar>
-    public var inputBias, updateBias, forgetBias, outputBias: Tensor<Scalar>
+    public var W_i, W_u, W_f, W_o: Tensor<Scalar>
+    public var b_i, b_u, b_f, b_o: Tensor<Scalar>
 
     @noDerivative public var stateShape: TensorShape {
         return TensorShape([1, inputWeight.shape[1]])
```

Comment:
I think this line doesn't compile, and should be changed to:

```suggestion
        return TensorShape([1, W_i.shape[1]])
```